provers/evm-prover/src/main.rs

```diff
@@ -133,7 +147,7 @@ impl Prover for ProverService {
     async fn info(&self, _request: Request<InfoRequest>) -> Result<Response<InfoResponse>, Status> {
         let response = InfoResponse {
             state_membership_verifier_key: "".to_string(),
-            state_transition_verifier_key: "".to_string(),
+            state_transition_verifier_key: hex::encode(self.aggregator_vkey.bytes32()),
```

Comment:
[blocking] the surrounding hex::encode isn't necessary

I added some debug logs to demonstrate

```
Got info request
Aggregator vkey: "0x00c27ff869a8aa1aeb631ae256639053c177d1863841815d12bd6e294db26283"
Aggregator vkey hex encoded: "307830306332376666383639613861613161656236333161653235363633393035336331373764313836333834313831356431326264366532393464623236323833"
```
```suggestion
            state_transition_verifier_key: self.aggregator_vkey.bytes32(),
```

Reply:
aghhhh